### PR TITLE
Removed aphrodite dependency

### PR DESCRIPTION
**Pull request by randamit123**
I removed the aphrodite dependency and replaced it with regular CSS modules so that we can transition to tailwind later on

### CODE REVIEW

**Comment by randamit123** [comment]
Changes made:
- Removed the aphrodite dependency
- Reconfigured all the components (4) that relied on it
- Added 4 CSS module files instead of aphrodite stylesheets
- Took out Instagram from socials

File: components/Curated.react.tsx [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
-import { StyleSheet, css } from "aphrodite";
 import { useMediaQuery } from "react-responsive";
 import SaferLink from "./SaferLink.react";
+import styles from "../styles/modules/Curated.react.module.css";
 
 const VideoThumbnail = ({ videoID, title }) => {
   const [isModalOpen, setIsModalOpen] = useState(false);
-  const dialogRef = useRef<HTMLDialogElement | null>(null);
-  const dialogIFrameRef = useRef<HTMLIFrameElement | null>(null);
+  const dialogRef = useRef(null);
```
Comment (chalkedgoose) [inline components/Curated.react.tsx]: I don't see how removing these type annotations is a good thing.

File: components/Curated.react.tsx [hunk 2]
```diff
@@ -92,9 +96,21 @@ type Props = Readonly<{
 }>;
 
 const Curated = ({ curated, className }: Props) => (
-  <Container className={`${css(styles.container)} ${className}`}>
+  <Container className={`${styles.container} ${className}`} >
     <h2>Community Curated Videos</h2>
-    <div className={css(styles.contentRow)}>
+    <div
+      style={{
+        display: "flex",
```
Comment (chalkedgoose) [inline components/Curated.react.tsx]: These styles should be in the css module.

File: components/NavbarMain.react.tsx [hunk 3]
```diff
@@ -43,12 +42,10 @@ const NavbarMain = (props: TProps) => {
     <React.Fragment>
       <Navbar
         sticky="top"
-        className="navbar-main py-2 justify-content-center"
-        bg="dark"
-        variant="dark"
+        className={`${styles["navbar-main"]} py-2 justify-content-center`}
```
Comment (chalkedgoose) [inline components/NavbarMain.react.tsx]: We can rename classes if they make it harder to work with css modules. 
I'd prefer ```navbarMain``` over this.

File: components/NavbarMain.react.tsx [hunk 4]
```diff
@@ -43,12 +42,10 @@ const NavbarMain = (props: TProps) => {
...
         expand="lg"
       >
-        <div className={`${css(navStyles.navElements)} w-100`}>
+        <div className={`${styles["navElements"]} w-100`}>
```
Comment (chalkedgoose) [inline components/NavbarMain.react.tsx:48]: same

File: components/NavbarMain.react.tsx [hunk 5]
```diff
@@ -84,7 +81,7 @@ const NavbarMain = (props: TProps) => {
                     />
                   ))}
                   <DiscordLink
-                    className="mx-2"
+                    className={`${styles["discord-link"]} mx-2`}
```
Comment (chalkedgoose) [inline components/NavbarMain.react.tsx]: same as navbar main issue

File: components/footer.jsx [hunk 6]
```diff
@@ -76,47 +73,28 @@ const Footer = () => {
...
-    display: "inline-block",
-    ":hover": {
-      filter: "invert(0.65)",
-    },
-  },
-});
-
 export default Footer;
+
+/*
```
Comment (chalkedgoose) [inline components/footer.jsx]: We don't need these comments worse case scenario you pin or open an new github issue with 
a due date on the action.

File: lib/links.js [hunk 7]
```diff
@@ -7,15 +7,14 @@ export function getNavLinks() {
...
-    { name: "Delia Garbacea", path: "https://www.deanza.edu/faculty/garbaceadelia/", external: true },
   ];
 }
 
 export function getCommunityLinks() {
   return [
     { path: "https://github.com/DevelopersGuild", logo: "/assets/icons/github.svg" },
     { path: "https://www.facebook.com/officialdevelopersguild", logo: "/assets/icons/facebook.svg" },
-    { path: "https://www.instagram.com/da.devguild", logo: "/assets/icons/instagram.svg" },
+    /*{ path: "https://www.instagram.com/da.devguild", logo: "/assets/icons/instagram.svg" }, Add instagram back when we createa an account*/
```
Comment (chalkedgoose) [inline lib/links.js]: Same issue here leave a todo comment with a github issue link or 
just make an issue.

File: pages/_app.js [hunk 8]
```diff
@@ -1,15 +1,14 @@
 import React from "react";
-import { StyleSheet } from "aphrodite";
 import "../styles/globals.css";
 import Layout from "../components/Layout";
 import 'bootstrap/dist/css/bootstrap.css';
 
 // Rehydrate to ensure that the client doesn't duplicate styles
```
Comment (chalkedgoose) [inline pages/_app.js]: Just delete the entirety of this block instead of a comment

**Comment by chalkedgoose** [comment]
It would be great to clean up our CSS code a bit more with the use of some structure or methodology 
one I see used a lot is BEM we should use it for our modules as well. https://en.bem.info/methodology/css/

**Review by chalkedgoose — CHANGES_REQUESTED** [review]
Left some feedback feel free to ping me on Discord or 
in one of these threads.

**Comment by chalkedgoose** [comment]
As an additional note it is generally a good idea to keep pull requests related to one set of changes like for example 
removing aphrodite. 

I noticed some other changes related to removing documents and changing the IG handle which 
are fine but should have been in a seperate PR. This makes reviewing code way faster and pleasant for the 
reviewer and increases the number of PRs on your Github account win win.

**Review by chalkedgoose — APPROVED** [review]
:shipit:

File: components/Curated.react.tsx (hunk 1 above)
Comment (charlescohanlon) [reply components/Curated.react.tsx]: lol

**Pull request merged**